spi/data-plane/data-plane-spi/src/main/java/org/eclipse/edc/connector/dataplane/spi/store/DataPlaneStore.java

```diff
@@ -14,44 +14,60 @@
 
 package org.eclipse.edc.connector.dataplane.spi.store;
 
+import java.util.Arrays;
+
 /**
  * Stores states of data flow requests.
  */
 public interface DataPlaneStore {
 
+    /**
+     * Mark the process as received.
+     */
+    void received(String processId);
+
+    /**
+     * Mark the process as completed.
+     */
+    void completed(String processId);
+
+    /**
+     * Returns the state of the process. If a process is unknown, returns {@link State#NOT_TRACKED}.
+     */
+    State getState(String processId);
+
     /**
      * Defines data flow states.
      */
     enum State {
         /**
          * There is no record of the process, i.e. one has not benn previously received.
          */
-        NOT_TRACKED,
+        NOT_TRACKED(0),
```

Comment:
in many of the other enums, we do this in 100 increments, to allow "space" for potential future states. Maybe this makes sense here too?